pex/third_party/__init__.py

```diff
@@ -116,8 +116,11 @@ def iter_root_packages(self, relpath):
       yield package
 
   def _filter_names(self, relpath, pattern, group):
+    # We use '/' here instead of os.sep because the zip file format spec specifies that paths must use forward slashes.
+    # See section 4.4.17 of https://support.pkware.com/display/PKZIP/APPNOTE
+    prefix = self.prefix + ((relpath + '/') if relpath else '')
```

Comment:
I think there's an opportunity to refactor this code to hopefully make this more readable and to avoid string concatenation—`.format()` is preferred in the codebase.

```python
prefix_suffix = "{}/".format(relpath) if relpath else ""
pat = re.compile(r'^{}{}$'.format(self.prefix + prefix_suffix, pattern)
```

Please feel free to iterate on this if you can think of a better approach!

Reply:
Got it. Sorry, I got lazy and copy pasted what was there before instead of following the rule of leaving things better than you found them.

Let me know what you think of what I pushed.

Reply:
Looks good! Thanks.